src/toolbars/SnappingToolBar.cpp

```diff
@@ -0,0 +1,425 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
```

Comment:
The commit that adds this file needs two `#include` directives to change so that it builds.